samples/Sample_19_SlideMaster.php

```diff
@@ -54,7 +54,7 @@
     ->setVertical(Alignment::VERTICAL_BASE);
 $shape->setAutoFit(RichText::AUTOFIT_NORMAL);
 $textRun = $shape->createTextRun('01-02-2000')->getFont()->setSize(18);
-$shape->setPlaceHolder(new Placeholder(Placeholder::PH_TYPE_DATETIME))->getPlaceholder()->setIdx(10);
```

Comment:
@qmachard Why do you remove the index ? 

If you remove the Idx, why don't you remove the getPlaceholder() ?